tasks/tasks/paragraph_embeddings_processors.py

```diff
@@ -1,21 +1,166 @@
 from base_annotation import BaseProcessor
 from elasticsearch import Elasticsearch
-import os
 import time
-from pypdf import PdfReader
-import ocrmypdf
+import os
 from typing import List, Tuple
 import numpy as np
 from utils import get_rawfile
-from embedder_engine import embedder
-
+from abc import ABC, abstractmethod
+import tiktoken
+import re
+from openai.embeddings_utils import get_embeddings
 from settings import settings
+from pypdf import PdfReader
+import ocrmypdf
+
+# import logging
```

Comment:
These tasks are very noisy and slow down on the intense logging I/O. For now I haven't enabled logging here (default for many tasks).

We should look into using the `es` bulk API and try to enable logging, which should help with document processing visibility.